python/tidy/servo_tidy_tests/test_tidy.py

```diff
@@ -13,6 +13,7 @@
 from servo_tidy import tidy
 
 base_path = 'servo_tidy_tests/' if os.path.exists('servo_tidy_tests/') else 'python/tidy/servo_tidy_tests/'
+COMPILE_FAIL_PATH = '../../../tests/compiletest/plugin/compile-fail/'
```

Comment:
Done, btw now I get this warning:

warning: function is never used: `match_ty_unwrap`

Should I got ahead and delete match_ty_unwrap?

Seems like a function that could be useful in the future.